reddit_liveupdate/activity.py

```diff
@@ -18,51 +19,48 @@
 ACTIVITY_FUZZING_THRESHOLD = 100
 
 
+@baseplate_integration.with_root_span("job.liveupdate_activity")
 def update_activity():
     events = {}
     event_counts = collections.Counter()
-    event_ids = ActiveVisitorsByLiveUpdateEvent._cf.get_range(
-        column_count=1, filter_empty=False)
 
-    for event_id, is_active in event_ids:
-        count = 0
+    for chunk in utils.in_chunks(LiveUpdateEvent._all(), size=100):
+        context_ids = {"LiveUpdateEvent_" + ev._id: ev._id for ev in chunk}
+
+        try:
+            with c.activity_service.retrying() as svc:
+                infos = svc.count_activity_multi(context_ids.keys())
```

Comment:
I'm having difficulty figuring out what's going on here (and can't find `count_activity_multi` in the live update plugin or the activity service).  It seems like we're kicking off a count process for every live update thread?  Perhaps a comment would help.